tests/kernel/timer/timer_behavior/Kconfig

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2022 Intel Corporation
+
+mainmenu "Timer Behavior Test"
+
+source "Kconfig.zephyr"
+
+config TIMER_TEST_SAMPLES
+	int "The number of timer samples to gather for statistics"
+	default 10000
+   
+config TIMER_TEST_PERIOD
+	int "The number of microseconds to for the timer period"
+	default 1000
```

Comment:
Nothing wrong with specifying this in ms here, but when used at runtime you want to be converting this to an integer number of ticks before use and doing all the validation math relative to that number, not real time.  There are multiple platforms now (Nordic being the giant in the list) with hardware clocks that can't produce a 1ms delay exactly.